README.md

```diff
@@ -21,6 +21,7 @@ Currently, the following adapters are supported:
 - Snowflake
 - DuckDB
 - Trino (tested with Iceberg connector)
+- AWS Athena
```

Comment:
We may want to add a callout in the docs to distinguish between adapters we have CI checks for, and adapters we don't - we likely won't have an AWS athena account to use to run CI checks on each PR

Reply:
Agree, I think that we need to highlight what is tested automatically and not